src/main/java/org/jabref/gui/exporter/ExportCommand.java

```diff
@@ -133,19 +133,23 @@ private void export(Path file, FileChooser.ExtensionFilter selectedExtensionFilt
                     return null; // can not use BackgroundTask.wrap(Runnable) because Runnable.run() can't throw Exceptions
                 })
                 .onSuccess(save -> {
-                    LibraryTab.DatabaseNotification notificationPane = tabSupplier.get().getNotificationPane();
-                    notificationPane.notify(
-                            IconTheme.JabRefIcons.FOLDER.getGraphicNode(),
-                            Localization.lang("Export operation finished successfully."),
-                            List.of(new Action(Localization.lang("Reveal in File Explorer"), event -> {
-                                try {
-                                    NativeDesktop.openFolderAndSelectFile(file, preferences.getExternalApplicationsPreferences(), dialogService);
-                                } catch (IOException e) {
-                                    LOGGER.error("Could not open export folder.", e);
-                                }
-                                notificationPane.hide();
-                            })),
-                            Duration.seconds(5));
+                    boolean libraryTab = tabSupplier.get() != null;
```

Comment:
The boolean variable should be named `currentTabIsLibraryTab`.

@calixtus Should we introduce an `Optional` at the `get()`? - Or do we need some more sophisticated logic here? Maybe also @InAnYan